evmos/chain.json

```diff
@@ -162,15 +159,12 @@
       },
       {
         "name": "v16.0.0",
-        "tag": "v16.0.3",
+        "tag": "v16.0.4",
         "proposal": 265,
         "height": 18295000,
-        "recommended_version": "v16.0.3",
+        "recommended_version": "v16.0.4",
         "compatible_versions": [
-          "v16.0.0",
-          "v16.0.1",
-          "v16.0.2",
-          "v16.0.3"
+          "v16.0.4"
```

Comment:
The previous versions are still compatible with v16.0.4

Reply:
v16.0.4 is required for the v17.0.0 correct? Can a node operator successfully upgrade to v17.0.0 from v16.0.3?